src/Bindings/LuaState.cpp

```diff
@@ -319,12 +319,30 @@ void cLuaState::Create(void)
 
 void cLuaState::RegisterAPILibs(void)
 {
+	auto top = lua_gettop(m_LuaState);
 	tolua_AllToLua_open(m_LuaState);
+	ASSERT(top == lua_gettop(m_LuaState));
 	cManualBindings::Bind(m_LuaState);
+	ASSERT(top == lua_gettop(m_LuaState));
 	DeprecatedBindings::Bind(m_LuaState);
+	ASSERT(top == lua_gettop(m_LuaState));
 	cLuaJson::Bind(*this);
+	ASSERT(top == lua_gettop(m_LuaState));
 	luaopen_lsqlite3(m_LuaState);
+	if (top == lua_gettop(m_LuaState) - 1)
+	{
+		// lsqlite3 left the "sqlite3" table on the stack, pop it:
+		lua_pop(m_LuaState, 1);
+	}
+	ASSERT(top == lua_gettop(m_LuaState));
 	luaopen_lxp(m_LuaState);
+	this->LogStackValues("After lxp");
```

Comment:
Why this log message?
https://travis-ci.org/cuberite/cuberite/jobs/151335092#L2177